rust/src/db.rs

```diff
@@ -0,0 +1,47 @@
+use anyhow::anyhow;
+use anyhow::Context;
+use anyhow::Result;
+use sqlx::pool::PoolConnection;
+use sqlx::sqlite::SqliteConnectOptions;
+use sqlx::Sqlite;
+use sqlx::SqlitePool;
+use state::Storage;
+use std::path::Path;
+use std::sync::Mutex;
+use std::sync::MutexGuard;
+
+pub type SqliteConnection = PoolConnection<Sqlite>;
+
+/// Wallet has to be managed by Rust as generics are not support by frb
```

Comment:
nit: copy & paste error

Wallet -> SqlitePool

Reply:
thanks, I'll remove the type name from the comment in both cases, it's irrelevant!